### PR TITLE
Add cell input extension

### PR DESCRIPTION
**Pull request by aamatveev**

### CODE REVIEW

**Comment by djhvscf** [comment]
Please provide a [jsFiddle](https://github.com/wenzhixin/bootstrap-table/blob/develop/CONTRIBUTING.md#bug-reports) to show this PR working, thanks!

**Comment by aamatveev** [comment]
Hi.
http://jsfiddle.net/amatveef/5qztmj0w/

**Comment by djhvscf** [comment]
This looks awesome. Please add a README with some documentation.

**Comment by aamatveev** [comment]
hi. I added a README

File: src/extensions/cell-input/bootstrap-table-cell-input.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        }
+
+        $.each(this.columns, function (i, column) {
+            if (column.cellInputType === "text") {
+                that.$body.find('input[data-name="' + column.field + '"]')
+                .off('blur').on('blur', function (e) {
+                    var data = that.getData(),
+                        index = $(this).parents('tr[data-index]').data('index'),
+                        row = data[index],
+                        newValue = $(this).val();
```
Comment (Falseee) [inline src/extensions/cell-input/bootstrap-table-cell-input.js]: https://github.com/wenzhixin/bootstrap-table/pull/4218#issuecomment-475822706

**Comment by wenzhixin** [comment]
Need to move `README.md` to https://github.com/wenzhixin/bootstrap-table/tree/develop/site/docs/extensions

**Comment by djhvscf** [comment]
Moving

**Pull request merged**